homeassistant/components/onvif/parsers.py

```diff
@@ -239,6 +251,37 @@ async def async_parse_motion_region_detector(uid: str, msg) -> Event:
             "binary_sensor",
             "motion",
             None,
+            msg.Message._value_1.Data.SimpleItem[0].Value in ["1", "true"],
+        )
+    except (AttributeError, KeyError):
+        return None
+
+
+@PARSERS.register("tns1:RuleEngine/LineDetector/Crossed")
```

Comment:
This ONVIF event has no state. Meaning that the event will only be sent once when a line is crossed, and will not be sent again with a `false` value.  That means with this implementation, the binary sensor would start as off, then be stuck `on` after the very first line crossing event.

See Annex A.2 of the Analytics Service Specification here: https://www.onvif.org/specs/srv/analytics/ONVIF-Analytics-Service-Spec.pdf

Reply:
In that case, maybe it could be represented as a button?

https://www.home-assistant.io/integrations/button/

I am not sure if this would be the right approach but it seems to fit what you describe.

Reply:
The button entity is not used to represent events coming from a device. It's used to call a method on the device without parameters, like a simple service.

Reply:
Events that doesn't represent state should just be fired as custom events in Home Assistant. Ie not be represented by any entity.

Reply:
Thanks @MartinHjelmare.  Ok, I suggest a short term solution is to handle this with a shorter warning message in the log saying that this is not implemented (rather than the current long json dump).  

Possibly the json dump could be in a debug log message.

@hunterjm would you agree with this approach?

Reply:
I think it would be fine to just change the log level of the existing message from warning to info.

Reply:
@hunterjm Ok, done.